index.html

```diff
@@ -0,0 +1,150 @@
+            </div>
+
+            <div class="formElment">
+                <label for="Email">Email</label>
+                <input 
+                type="email"
+                name="email"
+                placeholder="Enter your Email"
+                >
+            </div>
+
+            <div class="formElment">
+                <label for="Age">Age (optional)</label>
+                <input 
+                type="number"
+                name="Age"
+                placeholder="Age"
+                >
+            </div>
+            <div class="formElment">
+                <label>Which option best describes your current role?</label>
+                <select>
+                    <option disabledbyvalue>Select current role</option>
+                    <option value="Student">Student</option>
+                    <option value="job">Full Time Job</option>
+                    <option value="learner">Full Time Learner</option>
+                    <option value="nothing">Perfer not to say</option>
+                    <option value="Other">Other</option>
+                </select>
+            </div>
+
+            <div class="formElment">
+                <label for="recommend">Would you recommend freeCodeCamp to a friend?</label>
+
+                <fieldset id="recommend" style="border: 0;">
+                    <div>
+                        <label for="definately">
+                            <input id="definately" type="radio" value="definately" name="recommend">
+                            Definately
+                        </label> 
+                    </div>
+                    <div>
+                        <label for="maybe">
+                            <input id="maybe"  type="radio" value="maybe" name="recommend">
+                            Maybe
+                        </label>
+                    </div>
+                    <div>
+                        <label for="not_sure">
+                            <input id="not_sure" type="radio" value="not_sure" name="recommend">
+                            Not sure
+                        </label>
+                    </div>
+                </fieldset>
+                    
+            </div>
+            <div class="formElment">
+                <label>What is your favorite feature of freeCodeCamp?</label>
+                <select>
+                    <option disabledbyvalue>Select an option</option>
+                    <option value="Student">Challenges</option>
+                    <option value="job">Projects</option>
+                    <option value="learner">Community</option>
+                    <option value="nothing">Open source</option>
+                </select>
+            </div>
+
+            <div class="formElement">
+                <label for="">What would you like to see improved? </label>
+                <div>
+                    <input type="checkbox" name="" id="">
+                    Front-end Projects
+                </div>
```

Comment:
I suggest you use the unordered list for clarity. Use <ul> instead of the <div>. It's a better practice to use the <ul>

Reply:
- [ ] Good job on communicating the issue with a suggestion, this comment isn't formatted properly and does not point out what is used instead of the `<ul>` tag, kindly rephrase and format it highlighting the issue for the reviewee.